autoapi/templates/python/function.rst

```diff
@@ -1,6 +1,6 @@
 {%- if obj.display %}
 
-.. function:: {{ obj.name }}({{ obj.args|join(',') }})
+.. function:: {{ obj.short_name }}({{ obj.args|join(',') }})
```

Comment:
What in particular are you worried about? I tested it at the module level and inside a class.